src/test/java/se/lth/cs/connect/ITUserAPI.java

```diff
@@ -188,6 +188,12 @@ public void testDeleteEmptyCollections() throws JsonParseException, UnsupportedE
 		jp = res.jsonPath();
 		String entryID = jp.getString("id");
 		
+		//Simple check to confirm that entry was created.
```

Comment:
Are you sure that this is necessary? You are already checking the status code of the request where you create the entry.